utils/firestore.js

```diff
@@ -1866,7 +1866,73 @@ const verifyUsersEmailOrPhone = async (req) => {
             return false;
         }
     }
-}
+
+    const updateUsersCurrentEmail = async (email, uid) => {    
```

Comment:
I'm curious what @we-ai thinks as well, but with how similar these four functions are I would rather see these combined into one function. This would also make the code that calls this function "simpler".

Reply:
I combined Update current email & phone functions together. I prefer leaving replace phone & email separate. As the parameters to replace email & phone are different from each other.